Parser.py

```diff
@@ -161,16 +163,31 @@ def dict_2_md(input_list,file):
 
 
 def main():
-    #Command line argument 1, ekl file to open, else open sample
-    log_file = sys.argv[1] if len(sys.argv) >= 2 else "sample.ekl"
+    parser = argparse.ArgumentParser(
+        description='Process SCT results.'
+                    ' This program takes the SCT summary and sequence files,'
+                    ' and generates a nice report in mardown format.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '--md', help='Output .md filename', default='result.md')
+    parser.add_argument(
+        'log_file', nargs='?', default='sample.ekl',
+        help='Input .ekl filename')
+    parser.add_argument(
+        'seq_file', nargs='?', default='sample.seq',
+        help='Input .seq filename')
+    parser.add_argument('find_key', nargs='?', help='Search key')
```

Comment:
Might make sense to move `find_key` function into another command line option down the line, to avoid confusion. It's useful, but don't want to make it seem needed. Looks good for now though.